src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -297,10 +297,11 @@ configurationRegistry.registerConfiguration({
 			'default': true,
 			'description': localize('notebook.enableDoubleClickEdit', "Enable double click to edit for text cells in notebooks")
 		},
-		'notebook.setRichTextViewByDefault': {
-			'type': 'boolean',
-			'default': true,
-			'description': localize('notebook.setRichTextViewByDefault', "Set Rich Text View mode by default for text cells")
+		'notebook.defaultTextEditMode': {
+			'type': 'string',
+			'enum': [TextCellEditModes.RichText, TextCellEditModes.SplitView, TextCellEditModes.Markdown],
```

Comment:
Enum values should have localized descriptions : e.g. https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts#L37